src/main/java/com/ecore/roles/web/dto/MembershipDto.java

```diff
@@ -28,20 +22,17 @@ public class MembershipDto {
     private UUID id;
 
     @JsonProperty
-    @Valid
```

Comment:
@ NotNull can be used for mandatory field validation, @ Valid can be used for complex types.